packages/server/modules/workspaces/graph/resolvers/permissions.ts

```diff
@@ -0,0 +1,19 @@
+import { Resolvers } from '@/modules/core/graph/generated/graphql'
+import { Authz } from '@speckle/shared'
+
+export default {
+  Workspace: {
+    permissions: (parent) => ({
+      workspaceId: parent.id
+    })
+  },
+  WorkspacePermissionChecks: {
+    canCreateProject: async (parent, _args, ctx) => {
+      const canCreateProject = await ctx.authPolicies.workspace.canCreateProject({
```

Comment:
I already exposed the new policy through the GQL API, so when this is merged it should be working correctly